app/src/main/java/de/westnordost/streetcomplete/quests/road_surface/RoadSurfaceConfig.java

```diff
@@ -10,4 +10,12 @@
 			"track", "road",
 			/*"service", */ // this is too much, and the information value is very low
 	};
+
+	static final String[] ALL_ROADS = {
+			"trunk","trunk_link","motorway","motorway_link",
+			"primary", "primary_link", "secondary", "secondary_link", "tertiary", "tertiary_link",
+			"unclassified", "residential", "bicycle_road", "living_street", "pedestrian",
```

Comment:
https://github.com/westnordost/StreetComplete/issues/332 (it is also in unmodified SC)